dask_geopandas/core.py

```diff
@@ -88,7 +92,19 @@ def crs(self):
     @crs.setter
     def crs(self, value):
         """Sets the value of the crs"""
-        raise NotImplementedError
+        self.dask = self.set_crs(value).dask
```

Comment:
I think that you will also need to update some other state like `._name`

Reply:
Yep turns out I needed `_meta`